client/Configurator.c

```diff
@@ -546,7 +547,7 @@ static void security(struct Allocator* tempAlloc, List* conf, struct Log* log, s
     int chroot = 1;
     int setupComplete = 1;
     int setuser = 1;
-    if (Defined(win32)) {
+    if (Defined(win32) || Defined(__MINGW64__)) {
```

Comment:
Can we detect mingw in make.js and define `win32` whether we're 32 bit or 64 bit ?